noir-projects/noir-contracts/contracts/parent_contract/src/main.nr

```diff
@@ -242,4 +242,51 @@ contract Parent {
         // Call the target private function
         context.static_call_public_function(target_contract, target_selector, args);
     }
+
+    use dep::aztec::test::{helpers::{cheatcodes, test_environment::TestEnvironment}};
+    use dep::aztec::protocol_types::storage::map::derive_storage_slot_in_map;
+    use dep::aztec::note::note_getter::{MAX_NOTES_PER_PAGE, view_notes};
+    use dep::aztec::note::note_viewer_options::NoteViewerOptions;
+    use dep::child_contract::Child;
+    use dep::value_note::value_note::ValueNote;
+
+    #[test]
+    fn test_private_call() {
+        // Setup env, generate keys
+        let mut env = TestEnvironment::new();
+        let owner = env.create_account();
+        // Initial address, since we don't have to deploy Parent in order to test it
+        let parent_contract_address = cheatcodes::get_contract_address();
```

Comment:
Yeah, this is weird. Basically we have no real reason to actually deploy `ParentContract`, since it has no initializer. We just need a "base" contract address from where we can perform the calls...but just realized that's why I implemented `Contract::interface()`. Fixing!